manual/core/native_protocol/README.md

```diff
@@ -121,7 +133,7 @@ If you want to see the details of mixed cluster negotiation, enable `DEBUG` leve
   in the face of schema changes
 
 [protocol spec]: https://github.com/datastax/native-protocol/tree/1.x/src/main/resources
-[driver3]: https://docs.datastax.com/en/developer/java-driver/3.5/manual/native_protocol/
+[driver3]: https://docs.datastax.com/en/developer/java-driver/latest/manual/native_protocol/
```

Comment:
This takes us to a 404 today.

Reply:
The page moved to the developer section:

https://docs.datastax.com/en/developer/java-driver/latest/manual/developer/native_protocol/

Reply:
@adutra but we still shouldn't use `latest` in here as we want that link (i.e. `[driver3]`) to go to the latest 3.x version (i.e. `3.10` as of this writing) of docs. Hopefully I'm not missing anything!

Reply:
@msmygit you're right. I forgot that the 3.x and 4.x links are the same, so `latest` brings you to the most recent 4.x docs. We do want this link to be the latest 3.x, which is 3.10. And we'll have to remember to catch this one when we release new 3.x versions.....

Reply:
Sorry my bad. I didn't understand what the real issue was.